### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/cert-read-role-tf-module.git cert-read-role-tf-module
    cd cert-read-role-tf-module
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS README.md examples/basic_usage/README.md examples/basic_usage/main.tf examples/basic_usage/outputs.tf examples/basic_usage/variables.tf main.tf outputs.tf variables.tf 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👓

File: README.md [hunk 1]
```diff
@@ -59,21 +59,34 @@ This meta-role requires a permission policy similar to the following:
...
 
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
-| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored | `any` | n/a | yes |
-| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_path>/site.example.com/* | `string` | `live` | no |
+| account\_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
+| cert\_bucket\_name | The name of the AWS S3 bucket where certificates are stored | `any` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| account\_ids | AWS account IDs that are to be allowed to assume the role. | `list(string)` | `[]` | no |
| cert\_bucket\_name | The name of the AWS S3 bucket where certificates are stored. | `any` | n/a | yes |
```

File: README.md [hunk 2]
```diff
@@ -59,21 +59,34 @@ This meta-role requires a permission policy similar to the following:
...
+| cert\_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert\_path>/site.example.com/* | `string` | `"live"` | no |
```
Comment (mcdonnnj) [inline README.md]: I'm not sure if this is actually useful or if it just confuses the description. I leave it to your judgement.

```suggestion
| cert\_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at "<cert\_path>/site.example.com/*". | `string` | `"live"` | no |
```

File: README.md [hunk 3]
```diff
@@ -59,21 +59,34 @@ This meta-role requires a permission policy similar to the following:
...
 | hostname | The FQDN corresponding to the certificate to be read (e.g. site.example.com) | `any` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| hostname | The FQDN corresponding to the certificate to be read (e.g. site.example.com). | `any` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 4]
```diff
@@ -8,8 +8,30 @@ followed by the `terraform apply` command.
...
+## Inputs ##
+
+No inputs.
+
 ## Outputs ##
 
 | Name | Description |
 |------|-------------|
-| role_arn | The ARN corresponding to the IAM role to be used for reading certificate data for the specified hostname |
+| role\_arn | The ARN corresponding to the IAM role to be used for reading certificate data for site.example.com |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md:37]:
```suggestion
| role\_arn | The ARN corresponding to the IAM role to be used for reading certificate data for "site.example.com". |
```

File: README.md [hunk 5]
```diff
@@ -59,21 +59,34 @@ This meta-role requires a permission policy similar to the following:
...
 
 ## Outputs ##
```
Comment (mcdonnnj) [inline README.md:92]: The output for this module should have a period at the end of the description.

**Review by mcdonnnj — APPROVED** [review]
LGTM with the usual description suggestions from me. The actual variables/outputs would need to be updated as well.

File: README.md (hunk 3 above)
Comment (jsf9k) [reply README.md]: See commit 62847c1.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: See commit 62847c1.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit 62847c1.  I don't think the additional quotes add anything so I omitted them.

File: README.md (hunk 5 above)
Comment (jsf9k) [reply README.md:92]: See commit 3550618.

File: examples/basic_usage/README.md (hunk 4 above)
Comment (jsf9k) [reply examples/basic_usage/README.md:37]: See commit 3550618.  Again, I don't think anything is added by the quotes so I omitted them.

File: README.md [hunk 6]
```diff
@@ -59,28 +59,41 @@ This meta-role requires a permission policy similar to the following:
...
 
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| account_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
-| cert_bucket_name | The name of the AWS S3 bucket where certificates are stored | `any` | n/a | yes |
-| cert_path | The path to the certificates in the AWS S3 bucket.  For example, the certificate files for site.example.com are expected to live at <cert_path>/site.example.com/* | `string` | `live` | no |
-| hostname | The FQDN corresponding to the certificate to be read (e.g. site.example.com) | `any` | n/a | yes |
+| account\_ids | AWS account IDs that are to be allowed to assume the role | `list(string)` | `[]` | no |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| account\_ids | AWS account IDs that are to be allowed to assume the role. | `list(string)` | `[]` | no |
```

**Review by mcdonnnj — APPROVED** [review]
One missing, but thanks for adjusting these descriptions.

File: examples/basic_usage/README.md (hunk 4 above)
Comment (mcdonnnj) [reply examples/basic_usage/README.md:37]: @jsf9k I don't see this change in the current version of the file. That commit also does not include this change. Did I miss something?

File: README.md (hunk 6 above)
Comment (jsf9k) [reply README.md]: Doh!  I don't know how I missed that.

See commit 3b20e3b.

File: examples/basic_usage/README.md (hunk 4 above)
Comment (jsf9k) [reply examples/basic_usage/README.md:37]: Ah, I see it now.  I am blind today.

I think putting `site.example.com` there is misleading.  How about "The ARN corresponding to the IAM role to be used for reading certificate data for the host whose name is specified in the hostname input variable."?

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: See commit 3b20e3b.

File: examples/basic_usage/README.md (hunk 4 above)
Comment (mcdonnnj) [reply examples/basic_usage/README.md:37]: That works for me.

Comment (jsf9k) [reply examples/basic_usage/README.md:37]: See commit 850648c.

**Pull request merged**